content/intro-to-storybook/vue/fr/screen.md

```diff
@@ -0,0 +1,214 @@
+</div>
+
+## Fournir du contexte aux histoires
+
+La bonne nouvelle est qu'il est facile de fournir un magasin Vuex au composant `PureInboxScreen` dans une histoire! Nous pouvons créer un nouveau magasin dans notre fichier d'histoire et le transmettre comme contexte de l'histoire:
+
+```javascript
+//src/components/PureInboxScreen.stories.js
+import Vue from 'vue';
+import Vuex from 'vuex';
+import PureInboxScreen from './PureInboxScreen.vue';
+import { action } from '@storybook/addon-actions';
+import { defaultTasksData } from './PureTaskList.stories';
+Vue.use(Vuex);
+export const store = new Vuex.Store({
+  state: {
+    tasks: defaultTasksData,
+  },
+  actions: {
+    pinTask(context, id) {
+      action('pinTask')(id);
+    },
+    archiveTask(context, id) {
+      action('archiveTask')(id);
+    },
+  },
+});
+export default {
+  title: 'PureInboxScreen',
+  excludeStories: /.*store$/,
+};
+export const Default = () => ({
+  components: { PureInboxScreen },
+  template: `<pure-inbox-screen/>`,
+  store,
+});
+// liste de tâches avec les tâches épinglées
+export const error = () => ({
+  components: { PureInboxScreen },
+  template: `<pure-inbox-screen :error="true"/>`,
+});
+```
+
+Des approches similaires existent pour fournir un contexte simulé pour d'autres bibliothèques de données, telles qu'[Apollo](https://www.npmjs.com/package/apollo-storybook-decorator), [Relay](https://github.com/orta/react-storybooks-relay-container) et autres.
+
+En parcourant les états dans Storybook, il est facile de tester que nous avons fait cela correctement:
+
+<video autoPlay muted playsInline loop >
+
+  <source
+    src="/intro-to-storybook/finished-inboxscreen-states.mp4"
+    type="video/mp4"
+  />
+</video>
+
+## Le développement basé sur les composants (Component-Driven Development)
+
+Nous avons commencé par le composant le plus bas avec `Task`, puis nous sommes passés à `TaskList`, maintenant nous sommes ici avec une interface utilisateur à écran complet. Notre `InboxScreen` contient un composant de conteneur imbriqué et comprend des histoires d'accompagnement.
+
+<video autoPlay muted playsInline loop style="width:480px; height:auto; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/component-driven-development-optimized.mp4"
+    type="video/mp4"
+  />
+</video>
+
+[**Le développement basé sur les composants (Component-Driven Development)**](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) vous permet d'augmenter progressivement la complexité à mesure que vous montez dans la hiérarchie des composants. Parmi les avantages figurent un processus de développement plus ciblé et une couverture accrue de toutes les permutations possibles de l'interface utilisateur. En bref, CDD vous aide à créer des interfaces utilisateur de meilleure qualité et plus complexes.
```

Comment:
@jeanphibaconnais sorry for not addressing this earlier, but when you're able, please adjust the link "Le développement basé sur les composants (Component-Driven Development)" to point at https://www.componentdriven.org/

Reply:
no problem @jonniebigodes 😄  it's modified